docarray/array/storage/qdrant/getsetdel.py

```diff
@@ -0,0 +1,122 @@
+from typing import List, Iterable, Iterator, TYPE_CHECKING
+
+import scipy.sparse
+from qdrant_client import QdrantClient
+from qdrant_openapi_client.exceptions import UnexpectedResponse
+from qdrant_openapi_client.models.models import PointIdsList, PointsList, ScrollRequest
+
+from docarray import Document
+from docarray.array.storage.base.getsetdel import BaseGetSetDelMixin
+
+if TYPE_CHECKING:
+    from docarray.types import ArrayType
+
+
+class GetSetDelMixin(BaseGetSetDelMixin):
+
+    @property
+    def client(self) -> QdrantClient:
+        raise NotImplementedError()
+
+    @property
+    def serialization_config(self) -> dict:
+        raise NotImplementedError()
+
+    @property
+    def n_dim(self) -> int:
+        raise NotImplementedError()
+
+    @property
+    def collection_name(self) -> str:
+        raise NotImplementedError()
+
+    @property
+    def scroll_batch_size(self) -> int:
+        raise NotImplementedError()
```

Comment:
are these functions are necessary? I found they are implemented in `StorageMixins`